scilpy/reconst/afd_along_streamlines.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from dipy.data import get_sphere
+import numpy as np
+from scipy.special import lpn
+
+from scilpy.reconst.utils import find_order_from_nb_coeff, get_b_matrix
+from scilpy.tractanalysis.grid_intersections import grid_intersections
+
+
+def afd_map_along_streamlines(sft, fodf, fodf_basis, length_weighting):
+    """
+    Compute the mean Apparent Fiber Density (AFD) and mean Radial fODF
+    (radfODF) maps along a bundle.
+
+    Parameters
+    ----------
+    sft : StatefulTractogram
+        StatefulTractogram containing the streamlines needed.
+    fodf : np.ndarray
+        fODF with shape (X, Y, Z, #coeffs)
+        coeffs depending on the sh_order
+    fodf_basis : string
+        Has to be descoteaux07 or tournier07
+    length_weighting : bool
+        If set, will weigh the AFD values according to segment lengths
+
+    Returns
+    -------
+    afd_sum : np.array
+        AFD map (weighted if length_weighting)
+    rd_sum : np.array
+        rdAFD map (weighted if length_weighting)
+    """
+
+    sft.to_vox()
+
```

Comment:
Max said he is not getting the same results as before, could be that you are missing `sft.to_corner()`

It all depends on the behaviour of `grid_intersections`
What was the function being used in the old scilpy ?